src/repositories/access-log.repository.ts

```diff
@@ -1,19 +1,20 @@
 import sql from 'mssql'
-import db from '../db'
 import { FastifyBaseLogger } from 'fastify'
 
-const DB_NAME = 'PCM'
-
 export default class AccessLog {
   schema: string = '[audit].'
   _logger: FastifyBaseLogger
+  _pool: sql.ConnectionPool
 
-  constructor(logger: FastifyBaseLogger) { this._logger = logger }
+  constructor(logger: FastifyBaseLogger, pool: sql.ConnectionPool) {
+    this._logger = logger
+    this._pool = pool
+  }
 
   async get(user_id?: string) {
-    const r = new sql.Request(await db.get(DB_NAME))
+    const r = new sql.Request(this._pool)
     r.input('user_id', sql.Int, user_id)
-    this._logger.debug({ sqlParam: { user_id }, sqlDb: DB_NAME, sqlSchema: this.schema, sqlProc: '[uspGetAccessLogs]' }, 'running procedure')
```

Comment:
why remove db name? Cuz because of only 1 db being used?

Reply:
This variable is no longer accessible in the repository